apps/antalmanac/src/components/RightPane/SectionTable/SectionTableBody.tsx

```diff
@@ -49,7 +48,31 @@ const styles: Styles<Theme, object> = (theme) => ({
     },
     tr: {
         '&.addedCourse': {
-            backgroundColor: isDarkMode() ? '#b0b04f' : '#fcfc97',
+            // Additional specificity is needed so addedCourse takes precendence when timingWarning is also true
+            '&:nth-of-type(n)': {
+                background: isDarkMode() ? '#b0b04f' : '#fcfc97',
+            },
+        },
+        // Input on the stripe colors (currently red) would be appreciated
+        '&.scheduleConflict': {
+            '&:nth-of-type(odd)': {
+                background: `repeating-linear-gradient(
+                    45deg,
+                    rgba(244, 67, 54, 0.5),
+                    rgba(244, 67, 54, 0.5) 2px,
+                    ${theme.palette.action.hover} 2px,
+                    ${theme.palette.action.hover} 80px
+                )`,
+            },
+            '&:nth-of-type(even)': {
+                background: `repeating-linear-gradient(
```

Comment:
mfw the Cascading Style Sheets can't be cascaded to reuse some of these options 😭